src/main/java/com/timgroup/statsd/StatsDBlockingProcessor.java

```diff
@@ -0,0 +1,114 @@
+package com.timgroup.statsd;
+
+import java.io.IOException;
+import java.net.SocketAddress;
+import java.nio.ByteBuffer;
+import java.nio.channels.DatagramChannel;
+import java.nio.charset.Charset;
+import java.util.concurrent.Callable;
+
+import java.util.Queue;
```

Comment:
Note: Several imports are unused.

Reply:
We have to add linting to the CI - thank you for checking. I don't use an IDE, so..... 😊